addon/components/logo-maker.ts

```diff
@@ -0,0 +1,107 @@
+import { action } from '@ember/object';
+import Component from '@glimmer/component';
+import { tracked } from '@glimmer/tracking';
+
+interface LogoMakerArgs {
+  icon: string;
+  color: string;
+  onChange(icon: string, color: string): void;
+}
+
+export const DEFAULT_ICONS = [
+  'rabbit',
+  'star',
+  'heart',
+  'rocket-launch',
+  'money-bill',
+  'glass-whiskey-rocks',
+  'joystick',
+  'scroll',
+  'carrot',
+  'volleyball-ball',
+  'crown',
+  'chart-bar',
+  'users',
+  'saxophone',
+  'duck',
+  'university',
+  'tree-palm',
+  'trophy',
+  'pizza-slice',
+  'popcorn',
+  'chart-line',
+  'analytics',
+  'hat-wizard',
+  'chart-network',
+  'leaf',
+  'utensils',
+  'ufo',
+  'hat-winter',
+  'guitar',
+  'parachute-box',
+  'sheep',
+  'boot',
+  'shopping-basket',
+  'shopping-cart',
+  'shopping-bag',
+  'tags',
+  'meteor',
+  'globe-americas',
+  'globe-snow',
+  'user-hair',
+  'paper-plane-top',
+  'star',
```

Comment:
Waiting for feedback from Vero to switch out second star

Reply:
I think William had a solution for this but didn't have the time to update the designs.
Maybe also double check with him (if not already done ofc)